pocs/tests/conftest.py

```diff
@@ -12,12 +12,23 @@
 
 
 def pytest_addoption(parser):
+    parser.addoption("--hardware-test", action="store_true", default=False, help="Test with hardware attached")
```

Comment:
FWIW, doesn't seem related to the cleanup. Just nitpicking.

Reply:
Yes, good point. It will be need during the merge to make the PEAS tests work (or, rather, skip). I can pull out of here.